src/isar/mission_planner/echo_planner.py

```diff
@@ -50,7 +50,7 @@ def get_mission(self, mission_id: int) -> Mission:
             self.logger.error(msg)
             raise MissionPlannerError(msg) from e
 
-        mission: Mission = Mission(steps=[])
+        mission: Mission = Mission(tasks=[])
```

Comment:
Suggestion: Start with an empty list `tasks=[]` here, and then instantiate the `Mission` after the `tasks` list is populated

Reply:
Done